transactions/set-approved-id-list/dec-7/README.md

```diff
@@ -0,0 +1,73 @@
+# Add Approved Node Operators ID List
+
+> December 7th, 2021
+
+15 new external access nodes
+
+1 new Dapper Labs access node
+
+## Dapper Labs Builds
+
+```sh
+flow transactions build ./templates/set_approved_id_list_with_args.cdc \
```

Comment:
⚠️ This is using [`setApprovedList`](https://github.com/onflow/service-account/blob/kan/add-allow-list-node-ids/templates/set_approved_id_list_with_args.cdc), which overwrites the approved list. 

It should be using [this transaction](https://github.com/onflow/flow-core-contracts/blob/master/transactions/idTableStaking/admin/add_approved_nodes.cdc), which adds the new nodes to the existing list.